openvdb_wolfram/OpenVDBLink/Source/OpenVDBGrid.h

```diff
@@ -0,0 +1,438 @@
+// Copyright Contributors to the OpenVDB Project
+// SPDX-License-Identifier: MPL-2.0
+
+#ifndef OPENVDBLINK_OPENVDBGRID_HAS_BEEN_INCLUDED
+#define OPENVDBLINK_OPENVDBGRID_HAS_BEEN_INCLUDED
+
+#include "GlueTensors.h"
+
+#include "OpenVDBCommon.h"
+#include "LTemplate.h"
+
+#include <openvdb/openvdb.h>
+#include <openvdb/Types.h>
+#include <openvdb/tools/LevelSetUtil.h>
+
+#include <cmath>
+#include <ctime>
+#include <vector>
+
+using namespace openvdb;
+using namespace openvdb::tools;
+using namespace openvdbmma::types;
+using namespace openvdbmma::utils;
+
+
```

Comment:
Please consider wrapping this class in a namespace. The name `OpenVDBGrid` is generic enough that a user might use it, which may cause name collision if their library is linked against this sub-library.

Reply:
This has been added. It's now `openvdbmma::OpenVDBGrid`.